packages/components/avatar/src/avatar.ts

```diff
@@ -92,407 +26,133 @@ export class Avatar extends ScopedElementsMixin(LitElement) {
   /** @internal */
   static override styles: CSSResultGroup = styles;
 
-  /** Border width for calculations in the svg. */
-  #borderWidth = BORDER_WIDTH;
-
-  /** The avatar configuration settings from the current theme. */
-  #config?: AvatarConfig;
-
-  /** Observe the avatar width. */
-  #observer = new ResizeObserver(() => this.#checkOverflow());
-
-  /** Offset of the badge for calculations in the svg. */
-  #offset = OFFSET_CIRCLE;
+  /** Observe the size of various elements. */
+  #observer = new ResizeObserver(() => this.#onResize());
 
-  /** @internal The badge. */
-  @state() badge?: AvatarBadge;
+  /** The slotted badge element. */
+  badge?: Badge;
 
-  /**
-   * Text to show on the badge in the top right corner of the avatar.
-   * Be aware this text should not be more then a few characters.
-   * Typically this option is used to show a number, for example unread messages.
-   */
-  @property({ attribute: 'badge-text' }) badgeText?: string;
+  /** The clip-path cutout for the badge. */
+  @state() clipPath?: string;
 
   /** The initials that need to be displayed. If none are set they are determined based on the displayName .*/
   @property({ attribute: 'display-initials' }) displayInitials?: string;
 
   /** The name that needs to be displayed. */
   @property({ attribute: 'display-name' }) displayName?: string;
 
-  /** @internal State for when loading of the image has failed. */
-  @state() errorLoadingImage?: boolean;
-
-  /**
-   * The fallback to use when there is no user image present.
-   * @type {'initials' | 'image'}
-   */
-  @property() fallback?: AvatarFallbackType = 'initials';
-
   /** An optional URL that will be used for linking the display name. */
   @property() href?: string;
 
-  /** @internal The icon. */
-  @state() icon?: AvatarIcon;
-
-  /** @internal The image. */
-  @state() image?: AvatarImage;
-
   /** This hides the name when set to true. */
   @property({ type: Boolean, reflect: true, attribute: 'image-only' }) imageOnly?: boolean;
 
-  /**
-   * Used for the aria-label on the image. You can use `{{badgeText}}` in the string
-   * to have it replaced by the value set in the badgeText. For example to show "6 unread messages", where 6 is also shown in the badge.
-   */
-  @property() label = '';
+  /** @internal The initials, either explicitly via displayInitials, or implicitly via displayName. */
+  @state() initials = '';
 
   /** The url of the avatar image. */
   @property({ attribute: 'picture-url' }) pictureUrl?: string;
 
-  /**
-   * The size of the avatar.
-   * @type {'sm' | 'md' | 'lg' | 'xl' | '2xl' | '3xl'}
-   */
+  /** The size of the avatar. */
   @property({ reflect: true }) size: AvatarSize = 'md';
 
-  /**
-   * Optional user status to show.
-   * @type {'danger' | 'success' | 'warning' | 'accent' | 'neutral' | 'primary'}
-   */
-  @property({ reflect: true }) status?: UserStatus;
```

Comment:
Can you please change examples on the website as well? There are few with `status` left :)

<img width="949" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/123ce316-b308-4565-aec2-7cf3ebd43086">